{"created": 1740319412.25657, "duration": 0.08082818984985352, "exitcode": 1, "root": "/Users/tomhofkens/PycharmProjects/vbtoets", "environment": {}, "summary": {"failed": 10, "total": 10, "collected": 10}, "collectors": [{"nodeid": "", "outcome": "passed", "result": [{"nodeid": ".", "type": "Dir"}]}, {"nodeid": "lib", "outcome": "passed", "result": []}, {"nodeid": "test_vraag01.py", "outcome": "passed", "result": [{"nodeid": "test_vraag01.py::test_vraag01_case1", "type": "Function", "lineno": 9}, {"nodeid": "test_vraag01.py::test_vraag01_case2", "type": "Function", "lineno": 12}, {"nodeid": "test_vraag01.py::test_vraag01_case3", "type": "Function", "lineno": 15}, {"nodeid": "test_vraag01.py::test_vraag01_case4", "type": "Function", "lineno": 18}, {"nodeid": "test_vraag01.py::test_vraag01_case5", "type": "Function", "lineno": 21}]}, {"nodeid": "test_vraag02.py", "outcome": "passed", "result": [{"nodeid": "test_vraag02.py::test_vraag02_case1", "type": "Function", "lineno": 9}, {"nodeid": "test_vraag02.py::test_vraag02_case2", "type": "Function", "lineno": 12}, {"nodeid": "test_vraag02.py::test_vraag02_case3", "type": "Function", "lineno": 17}]}, {"nodeid": "test_vraag04.py", "outcome": "passed", "result": [{"nodeid": "test_vraag04.py::test_vraag04", "type": "Function", "lineno": 12}]}, {"nodeid": "test_vraag05.py", "outcome": "passed", "result": [{"nodeid": "test_vraag05.py::test_vraag05", "type": "Function", "lineno": 9}]}, {"nodeid": ".", "outcome": "passed", "result": [{"nodeid": "lib", "type": "Dir"}, {"nodeid": "test_vraag01.py", "type": "Module"}, {"nodeid": "test_vraag02.py", "type": "Module"}, {"nodeid": "test_vraag04.py", "type": "Module"}, {"nodeid": "test_vraag05.py", "type": "Module"}]}], "tests": [{"nodeid": "test_vraag01.py::test_vraag01_case1", "lineno": 9, "outcome": "failed", "keywords": ["test_vraag01_case1", "test_vraag01.py", "vbtoets", ""], "setup": {"duration": 0.00024107799981720746, "outcome": "passed"}, "call": {"duration": 0.00036728300619870424, "outcome": "failed", "crash": {"path": "/Users/tomhofkens/PycharmProjects/vbtoets/lib/utils.py", "lineno": 32, "message": "AssertionError"}, "traceback": [{"path": "test_vraag01.py", "lineno": 11, "message": "in test_vraag01_case1"}, {"path": "lib/utils.py", "lineno": 32, "message": "in check_exact_match"}], "stdout": "\n\n\u001b[91mE        De functie 'naar_links1([8, 2, 2, 0], 0)' geeft niet het verwachte resultaat.\u001b[0m\n\u001b[91mE        Verwacht: [8, 2, 2, 0]\u001b[0m\n\u001b[91mE        Gekregen: None\u001b[0m\n", "longrepr": "test_vraag01.py:11: in test_vraag01_case1\n    check_exact_match(vraag01, \"naar_links1\", ([8, 2, 2, 0], 0), [8, 2, 2, 0])\nlib/utils.py:32: in check_exact_match\n    assert False\nE   AssertionError"}, "teardown": {"duration": 0.00034285600122530013, "outcome": "passed"}}, {"nodeid": "test_vraag01.py::test_vraag01_case2", "lineno": 12, "outcome": "failed", "keywords": ["test_vraag01_case2", "test_vraag01.py", "vbtoets", ""], "setup": {"duration": 0.00014378500054590404, "outcome": "passed"}, "call": {"duration": 0.0002732290013227612, "outcome": "failed", "crash": {"path": "/Users/tomhofkens/PycharmProjects/vbtoets/lib/utils.py", "lineno": 32, "message": "AssertionError"}, "traceback": [{"path": "test_vraag01.py", "lineno": 14, "message": "in test_vraag01_case2"}, {"path": "lib/utils.py", "lineno": 32, "message": "in check_exact_match"}], "stdout": "\n\n\u001b[91mE        De functie 'naar_links1([8, 2, 2, 0], 3)' geeft niet het verwachte resultaat.\u001b[0m\n\u001b[91mE        Verwacht: [8, 2, 2, 0]\u001b[0m\n\u001b[91mE        Gekregen: None\u001b[0m\n", "longrepr": "test_vraag01.py:14: in test_vraag01_case2\n    check_exact_match(vraag01, \"naar_links1\", ([8, 2, 2, 0], 3), [8, 2, 2, 0])\nlib/utils.py:32: in check_exact_match\n    assert False\nE   AssertionError"}, "teardown": {"duration": 0.00020988800679333508, "outcome": "passed"}}, {"nodeid": "test_vraag01.py::test_vraag01_case3", "lineno": 15, "outcome": "failed", "keywords": ["test_vraag01_case3", "test_vraag01.py", "vbtoets", ""], "setup": {"duration": 0.0001580660027684644, "outcome": "passed"}, "call": {"duration": 0.0002640750026330352, "outcome": "failed", "crash": {"path": "/Users/tomhofkens/PycharmProjects/vbtoets/lib/utils.py", "lineno": 32, "message": "AssertionError"}, "traceback": [{"path": "test_vraag01.py", "lineno": 17, "message": "in test_vraag01_case3"}, {"path": "lib/utils.py", "lineno": 32, "message": "in check_exact_match"}], "stdout": "\n\n\u001b[91mE        De functie 'naar_links1([8, 2, 2, 0], 2)' geeft niet het verwachte resultaat.\u001b[0m\n\u001b[91mE        Verwacht: [8, 4, 0, 0]\u001b[0m\n\u001b[91mE        Gekregen: None\u001b[0m\n", "longrepr": "test_vraag01.py:17: in test_vraag01_case3\n    check_exact_match(vraag01, \"naar_links1\", ([8, 2, 2, 0], 2), [8, 4, 0, 0])\nlib/utils.py:32: in check_exact_match\n    assert False\nE   AssertionError"}, "teardown": {"duration": 0.00019495699962135404, "outcome": "passed"}}, {"nodeid": "test_vraag01.py::test_vraag01_case4", "lineno": 18, "outcome": "failed", "keywords": ["test_vraag01_case4", "test_vraag01.py", "vbtoets", ""], "setup": {"duration": 0.00012191499990876764, "outcome": "passed"}, "call": {"duration": 0.00027483899611979723, "outcome": "failed", "crash": {"path": "/Users/tomhofkens/PycharmProjects/vbtoets/lib/utils.py", "lineno": 32, "message": "AssertionError"}, "traceback": [{"path": "test_vraag01.py", "lineno": 20, "message": "in test_vraag01_case4"}, {"path": "lib/utils.py", "lineno": 32, "message": "in check_exact_match"}], "stdout": "\n\n\u001b[91mE        De functie 'naar_links1([8, 0, 2, 0], 2)' geeft niet het verwachte resultaat.\u001b[0m\n\u001b[91mE        Verwacht: [8, 2, 0, 0]\u001b[0m\n\u001b[91mE        Gekregen: None\u001b[0m\n", "longrepr": "test_vraag01.py:20: in test_vraag01_case4\n    check_exact_match(vraag01, \"naar_links1\", ([8, 0, 2, 0], 2), [8, 2, 0, 0])\nlib/utils.py:32: in check_exact_match\n    assert False\nE   AssertionError"}, "teardown": {"duration": 0.0001710920041659847, "outcome": "passed"}}, {"nodeid": "test_vraag01.py::test_vraag01_case5", "lineno": 21, "outcome": "failed", "keywords": ["test_vraag01_case5", "test_vraag01.py", "vbtoets", ""], "setup": {"duration": 0.00012058600259479135, "outcome": "passed"}, "call": {"duration": 0.00023501800023950636, "outcome": "failed", "crash": {"path": "/Users/tomhofkens/PycharmProjects/vbtoets/lib/utils.py", "lineno": 32, "message": "AssertionError"}, "traceback": [{"path": "test_vraag01.py", "lineno": 23, "message": "in test_vraag01_case5"}, {"path": "lib/utils.py", "lineno": 32, "message": "in check_exact_match"}], "stdout": "\n\n\u001b[91mE        De functie 'naar_links1([8, 2, 2, 0], 1)' geeft niet het verwachte resultaat.\u001b[0m\n\u001b[91mE        Verwacht: [8, 2, 2, 0]\u001b[0m\n\u001b[91mE        Gekregen: None\u001b[0m\n", "longrepr": "test_vraag01.py:23: in test_vraag01_case5\n    check_exact_match(vraag01, \"naar_links1\", ([8, 2, 2, 0], 1), [8, 2, 2, 0])\nlib/utils.py:32: in check_exact_match\n    assert False\nE   AssertionError"}, "teardown": {"duration": 0.00020199800201226026, "outcome": "passed"}}, {"nodeid": "test_vraag02.py::test_vraag02_case1", "lineno": 9, "outcome": "failed", "keywords": ["test_vraag02_case1", "test_vraag02.py", "vbtoets", ""], "setup": {"duration": 0.00033900700509548187, "outcome": "passed"}, "call": {"duration": 0.00018989400996360928, "outcome": "failed", "crash": {"path": "/Users/tomhofkens/PycharmProjects/vbtoets/lib/utils.py", "lineno": 23, "message": "TypeError: C() takes 0 positional arguments but 2 were given"}, "traceback": [{"path": "test_vraag02.py", "lineno": 11, "message": "in test_vraag02_case1"}, {"path": "lib/utils.py", "lineno": 23, "message": "in check_exact_match"}], "longrepr": "test_vraag02.py:11: in test_vraag02_case1\n    check_exact_match(vraag02, \"C\", (0, 0), 1)\nlib/utils.py:23: in check_exact_match\n    result = function(*args)\nE   TypeError: C() takes 0 positional arguments but 2 were given"}, "teardown": {"duration": 0.00014476399519480765, "outcome": "passed"}}, {"nodeid": "test_vraag02.py::test_vraag02_case2", "lineno": 12, "outcome": "failed", "keywords": ["test_vraag02_case2", "test_vraag02.py", "vbtoets", ""], "setup": {"duration": 0.00012701800733339041, "outcome": "passed"}, "call": {"duration": 0.00017420199583284557, "outcome": "failed", "crash": {"path": "/Users/tomhofkens/PycharmProjects/vbtoets/lib/utils.py", "lineno": 23, "message": "TypeError: C() takes 0 positional arguments but 2 were given"}, "traceback": [{"path": "test_vraag02.py", "lineno": 14, "message": "in test_vraag02_case2"}, {"path": "lib/utils.py", "lineno": 23, "message": "in check_exact_match"}], "longrepr": "test_vraag02.py:14: in test_vraag02_case2\n    check_exact_match(vraag02, \"C\", (1, 0), 1)\nlib/utils.py:23: in check_exact_match\n    result = function(*args)\nE   TypeError: C() takes 0 positional arguments but 2 were given"}, "teardown": {"duration": 0.00013859799946658313, "outcome": "passed"}}, {"nodeid": "test_vraag02.py::test_vraag02_case3", "lineno": 17, "outcome": "failed", "keywords": ["test_vraag02_case3", "test_vraag02.py", "vbtoets", ""], "setup": {"duration": 0.00012136199802625924, "outcome": "passed"}, "call": {"duration": 0.00016353599494323134, "outcome": "failed", "crash": {"path": "/Users/tomhofkens/PycharmProjects/vbtoets/lib/utils.py", "lineno": 23, "message": "TypeError: C() takes 0 positional arguments but 2 were given"}, "traceback": [{"path": "test_vraag02.py", "lineno": 19, "message": "in test_vraag02_case3"}, {"path": "lib/utils.py", "lineno": 23, "message": "in check_exact_match"}], "longrepr": "test_vraag02.py:19: in test_vraag02_case3\n    check_exact_match(vraag02, \"C\", (1, 1), 1)\nlib/utils.py:23: in check_exact_match\n    result = function(*args)\nE   TypeError: C() takes 0 positional arguments but 2 were given"}, "teardown": {"duration": 0.00019146800332237035, "outcome": "passed"}}, {"nodeid": "test_vraag04.py::test_vraag04", "lineno": 12, "outcome": "failed", "keywords": ["test_vraag04", "test_vraag04.py", "vbtoets", ""], "setup": {"duration": 0.00012480499572120607, "outcome": "passed"}, "call": {"duration": 0.0002854960039258003, "outcome": "failed", "crash": {"path": "/Users/tomhofkens/PycharmProjects/vbtoets/lib/utils.py", "lineno": 32, "message": "AssertionError"}, "traceback": [{"path": "test_vraag04.py", "lineno": 14, "message": "in test_vraag04"}, {"path": "lib/utils.py", "lineno": 32, "message": "in check_exact_match"}], "stdout": "\n\n\u001b[91mE        De functie 'integraal(<function f at 0x1112e8400>, 0, 1, 100)' geeft niet het verwachte resultaat.\u001b[0m\n\u001b[91mE        Verwacht: 0.33332500000000037\u001b[0m\n\u001b[91mE        Gekregen: 0.3283500000000004\u001b[0m\n", "longrepr": "test_vraag04.py:14: in test_vraag04\n    check_exact_match(vraag04, \"integraal\", (f, 0, 1, 100), 0.33332500000000037)\nlib/utils.py:32: in check_exact_match\n    assert False\nE   AssertionError"}, "teardown": {"duration": 0.000208198995096609, "outcome": "passed"}}, {"nodeid": "test_vraag05.py::test_vraag05", "lineno": 9, "outcome": "failed", "keywords": ["test_vraag05", "test_vraag05.py", "vbtoets", ""], "setup": {"duration": 0.000245324001298286, "outcome": "passed"}, "call": {"duration": 0.00018989200179930776, "outcome": "failed", "crash": {"path": "/Users/tomhofkens/PycharmProjects/vbtoets/lib/utils.py", "lineno": 55, "message": "TypeError: variantie() takes 0 positional arguments but 3 were given"}, "traceback": [{"path": "test_vraag05.py", "lineno": 11, "message": "in test_vraag05"}, {"path": "lib/utils.py", "lineno": 55, "message": "in check_approx_match"}], "longrepr": "test_vraag05.py:11: in test_vraag05\n    check_approx_match(vraag05, \"variantie\", ([1, 2, 3],[0.2, 0.5, 0.3],2.1), 0.49)\nlib/utils.py:55: in check_approx_match\n    result = function(*args)\nE   TypeError: variantie() takes 0 positional arguments but 3 were given"}, "teardown": {"duration": 0.0001638329995330423, "outcome": "passed"}}]}